win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5318,25 +5318,26 @@ ftGdiGetTextMetricsW(
             Status = STATUS_SUCCESS;
 
             IntLockFreeType();
+
+            Error = FT_Get_WinFNT_Header(Face, &Win);
```

Comment:
Maybe you could rearrange a bit these calls in a manner similar to here:
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/gdi/ntgdi/freetype.c;hb=dad3a0938afd502831a8e401aae2b6b5a7fc3c92#l3142
(doing FT_Get_WinFNT_Header and the two FT_Get_Sfnt_Table, and then do the error checks).

Reply:
Fixed.